script/rhcos_versions.sh

```diff
@@ -21,4 +22,4 @@ done
 
 sort -o ./rhcos_version_map ./rhcos_version_map 
 
-echo OpenShift to RHCOS version mapping is in 'rhcos_version_map'
+echo "OpenShift to RHCOS version mapping is in rhcos_version_map"
```

Comment:
No need for any quotation. Shell is very string friendly.

Reply:
Done.